### PR TITLE
Cleaner logging

### PR DESCRIPTION
**Pull request by lkarsten**
Attempt at reducing clutter in the debug log to make it easier to read.

### CODE REVIEW

File: src/rule.cc [hunk 1]
```diff
@@ -644,29 +641,27 @@ void Rule::executeActionsAfterFullMatch(Transaction *trans,
...
-                    "action: " + a->m_name);
-                    a->evaluate(this, trans, ruleMessage);
+                trans->debug(4, "Running (non-disruptive) action: " + a->m_name);
+                a->evaluate(this, trans, ruleMessage);
             }
             continue;
         }
         if (trans->m_rules->m_secRuleEngine == Rules::EnabledRuleEngine) {
-            trans->debug(4, "Running (disruptive) action: " + a->m_name);
+            trans->debug(4, "Running (disruptive)     action: " + a->m_name);
```
Comment (zimmerle) [inline src/rule.cc]: Keeping the disruptive vs. non-disruptive in the same col?

**Comment by zimmerle** [comment]
Hi @lkarsten,

Thank you for the patch. As a consequence of this changes, one of our test cases was failing. The **3rd test** of the **action-disruptive.json**. The test was based on debug log match, which was changed by your patch.

I've merged your patch and fixed the test case. Thank you ;)

File: src/rule.cc (hunk 1 above)
Comment (lkarsten) [inline src/rule.cc]: Yes! :)

**Comment by lkarsten** [comment]
Thanks for merging!

**Pull request closed without merging**